APE0.rst

```diff
@@ -0,0 +1,360 @@
+* a willingness to dedicate time to improving the Project,
+* a willingness to recruit, train, and nominate new team members.
+
+Voting Membership acknowledges sustained and valuable efforts that align well
+with the philosophy and the goals of the Astropy Project.
+
+Initial Membership
+^^^^^^^^^^^^^^^^^^
+Anyone who satisfies any of the following criteria at the date of acceptance of
+this APE will be invited to be in the first group of Voting Members:
+
+* has a named role in the project and has participated in an Astropy
+  coordination meeting in the last two years,
+* has a named role in the project and has regularly participated in Astropy
+  telecons (e.g., co-working telecon, spectroscopy sprint, Astropy Learn
+  telecon) in the last two years,
+* has commit rights to at least one repository in the astropy GitHub
+  organization and has actively used those commit rights in the last two years.
+
+Add New Voting Members
+^^^^^^^^^^^^^^^^^^^^^^
+Anyone can apply to become a Voting Member by providing evidence of meeting the
+requirements laid out in the Membership section above. Both self-nominations and
+nominations by others are allowed. The Voting Members are expected to make their
+decisions based on the candidate’s adherence to the membership criteria, above.
+The name of the nominee will be known to Voting Members but will not be shared
+outside Voting Members and the Coordination Committee unless the nominee becomes
+a Voting Member.
+
+The procedure for voting to add new Voting Members is:
+
+* The Coordination Committee and Ombudsperson receive each nomination, check
+  that it is factually accurate, that the nominated person accepts the
+  nomination, and that their record of community activity adheres to the Code of
+  Conduct.
+* Once a nomination is accepted by the Coordination Committee, the Coordination
+  Committee will initiate a vote among active Voting Members. The voting will be
+  open for at least two weeks and will conclude within four weeks of the
+  nomination being accepted. To successfully gain an appointment as a Voting
+  Member, the nominee must receive at least four positive votes, as long as that
+  constitutes a majority of those voting.
+* The candidate will be informed promptly at the close of voting by a
+  Coordination Committee member. If the vote is not affirmative, the
+  Coordination Committee will provide feedback to the nominee.
+
+Term and Active/Inactive Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members have no term or term limits. Voting Members who have stopped
+contributing are encouraged to declare themselves as "inactive.” Those who have
+not made any significant contribution for a long period may be asked to move
+themselves to the “inactive” category by the Coordination Committee. If no
+response is received, the Coordination Committee may automatically change a
+Voting Member’s status to inactive. To record and honor their contributions,
```

Comment:
What is a member never does anything for the project, except to reply to the Coco "I'm still active"? Should they be retired eventually? I'm not sure that's going to be a problem in practice, so I'm OK with not addressing it now, but I wanted to raise the possibility.

Reply:
In Debian, people who are no longer contribute, can request "Emeritus" status, which sounds more positive than "inactive". I would also propose a similar to Debian: If someone does not contribute within some period (1y?), the CC should try to reach them and ask whether they want to get this status.

Reply:
I also raised the desire to have the emeritus status for the roles here: https://github.com/astropy/astropy-project/issues/136#issuecomment-693708410, with the motivation of the roles page to reflect the actual reality.

Reply:
Good points - I agree "emeritus" is a better term regardless!

As for the "never does anything except reply", if I recall correctly the WG discussed the broader question of kicking people out and decided that it's probably better to let people be self-reporting, because we have to rely on that *anyway* in many cases. So someone who constantly replies "yes I am active" may actually have a good reason, and implicitly they are trustworthy be getting into the voting membership in the first place  (note that means not *everyone* has to trust them, but enough people to make it there in the first place).

That said, the removal clause is there for someone that a supermajority agrees is abusing that privledge. So the thinking is that this is how the case @hamogu described would be dealt with.

Reply:
(And I think emeritus status for *roles* is another interesting discussion to consider, but is not quite the same as emeritus status in a voting membership sense, since some people might be emeritus in one role but not another and therefore still active in the project as a whole.)

Reply:
Changed to emeritus